src/mumble/ManualPlugin.cpp

```diff
@@ -216,16 +216,10 @@ static void config(void *ptr) {
 
 	if (mDlg) {
 		mDlg->setParent(w, Qt::Dialog);
```

Comment:
We need to make sure that the Unhinge button is still available when we set the configuration window as parent.
```
if (!mDlg->qpbUnhinge->isEnabled)
	mDlg->qpbUnhinge->setEnabled(true);
```